streampark-console/streampark-console-service/src/main/resources/mapper/core/FlinkSqlMapper.xml

```diff
@@ -48,4 +48,9 @@
         where s.app_id = a.id
         and a.team_id = #{teamId}
     </select>
+
+    <select id="pageByAppId" resultType="org.apache.streampark.console.core.entity.FlinkSql" parameterType="java.lang.Long">
```

Comment:
Could the lines inline into the `lambdaquery` ?

Reply:
nice catch, change this to lambdaquery .